rpc/test/helpers.go

```diff
@@ -170,17 +179,71 @@ func NewTendermint(app abci.Application, opts *Options) *nm.Node {
 	if err != nil {
 		panic(err)
 	}
+
+	config.IPFS = cfg.DefaultIPFSConfig()
+	err = initIpfs(config, opts.loadIpfsPlugins, logger)
+	if err != nil {
+		panic(err)
+	}
 	node, err := nm.NewNode(config, pv, nodeKey, papp,
 		nm.DefaultGenesisDocProviderFunc(config),
 		nm.DefaultDBProvider,
 		nm.DefaultMetricsProvider(config.Instrumentation),
-		logger)
+		logger,
+		nm.IpfsPluginsWereLoaded(true),
+	)
 	if err != nil {
 		panic(err)
 	}
 	return node
 }
 
+func initIpfs(config *cfg.Config, loadPlugins bool, log log.Logger) error { // add counter part in ResetAllCmd
+	// init IPFS config with params from config.IPFS
+	// and store in config.IPFS.ConfigRootPath
```

Comment:
Same as above w.r.t comment position

Reply:
here it can be removed as the repos are temporary anyways: they only exist during the test-runs.